### ISSUE TITLE
woah - amazing

### ISSUE THREAD

**Issue by kieranjol** [amiaopensource/av-python-carpentry#2]
dang - i took a quick look and this is incredible. Awesome work!!

**Comment by bturkus** [comment]
it's very much a work-in-progress, but we'd love for you to review in a proper fashion after we get it together (which may not be before AMIA lol)

**Comment by kieranjol** [comment]
I'm so happy to!

**Comment by nkrabben** [comment]
Agreed, need to slow down on the amazing and increase the pedagogical rigor.

I'll keep this open until we need to file a giddyup - amazing issue.

**Comment by mercuryswitch** [comment]
This is exactly what I’ve been looking for! Thanks!
